pages/blog/june-2021-at-asyncapi.md

```diff
@@ -0,0 +1,183 @@
+
+## New CLI released
+
+The initial version of the [AsyncAPI CLI](https://github.com/asyncapi/cli) is finally here :rocket:.
+
+[Jorge Aguiar Martín](https://twitter.com/jotamusik) finished his hard work on initial setup of the CLI with first initial feature for AsyncAPI documents validation:
+
+<Figure
+  src="/img/posts/june-2021-at-asyncapi/cli.webp"
+  className="text-center"
+/>
+
+The next features are on their way. Feel free to join and work on it together with us.
+
+## Modelina supports Go
+
+Our model generation library is under heavy development. I want to explicitly mention one change among all the recent changes in [Modelina](https://github.com/asyncapi/modelina/releases). [Sergio Moya](https://github.com/smoya) enabled support for model generation for Go :muscle: It is a 4th language that is supported by the library.
+
+```js
+const { GoGenerator} = require("@asyncapi/modelina")
+
+const generator = new GoGenerator();
+
+const doc = {
+  $id: "Address",
+  type: "object",
+  properties: {
+    street_name:    { type: "string" },
+    city:           { type: "string", description: "City description" },
+    house_number:   { type: "number" },
+    marriage:       { type: "boolean", description: "Status if marriage live in given house" },
+    pet_names:      { type: "array", items: { type: "string" } },
+    state:          { type: "string", enum: ["Texas", "Alabama", "California", "other"] },
+  },
+  required: ["street_name", "city", "state", "house_number", "state"],
+};
+
+async function generate() {
+  const models = await generator.generate(doc);
+  models.forEach(function (model) {
+    console.log(model.result);
+  }); 
+}
+
+generate();
+
+//outputs
+
+/*
+// Address represents a Address model.
+type Address struct {
+  StreetName string
+  City string
+  HouseNumber float64
+  Marriage bool
+  PetNames []string
+  State *State
+}
+// State represents an enum of string.
+type State string
+*/
+```
+
+## React component vs HTML template and where are we
+
+[React component](https://github.com/asyncapi/asyncapi-react) is still under development towards 1.0.0 release. Keep in mind that we are already using release candidates in the [HTML template](https://github.com/asyncapi/html-template), and you can give it a try too:
+
+```
+npm install --save @asyncapi/react-component@v1.0.0-next.11
+```
+
+In June, a couple of release candidates were released. Most important to notice is a new standalone bundle that makes it super easy to reuse React component in [Angular](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/angular.md) and [Vue](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/vue.md) projects. In addition, we now provide also **esm** and **umd** modules. As a result, it is much easier to use React component with [Next.js](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/nextjs.md) projects.
```

Comment:
```suggestion
In June, a couple of release candidates were released. Most important to notice is a new standalone bundle that makes it super easy to reuse React component in [Angular](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/angular.md) and [Vue](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/vue.md) projects. In addition, we now provide not only **cjs** but also **esm** and **umd** modules. As a result, it is much easier to use React component with [Next.js](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/nextjs.md) projects. In addition, the component supports whole specification, except **discriminator**.
```

Reply:
can you change `In addition, the component supports whole specification, except **discriminator**` to `In addition, the component supports the whole specification, except **discriminator**`

Reply:
Changed :)